### DISCUSSION TITLE
Need advices about ordering Level Shifter boards (v1.1 VS v1.2)

### DISCUSSION THREAD

**Discussion by MoockyLoock** [darthcloud/BlueRetro#147] [Q&A] [answered]
Hello,
i'm preparing all my stuff to start building my own set of cables.
I have a few questions :
- there are 2 revisions of Level Shifter PCB in the Cables build instructions, but i can only find the V1.1 files in the hardware repository. Is the v1.2 not available for now (kinda beta version) ?
- could we use the V1.2 instead of v1.1 (if available) ?
- are the little electronic components on the bottom of the Level Shifter V1.2 the pull-up resistor ? If yes, what are their part number please ?

Thank you very much for your help.
And congratulation for this project !

**Comment by darthcloud** [comment, marked as answer]
The Kicad project is v1.2.

But I guess you means the pdf and gerber where not updated.

I updated both just now.

The new pad are for pull up resistor, so far it's only for PAL NES/SNES and PCE

**Comment by MoockyLoock** [comment]
Hey thanks you.
Yes i meant the gerber files to ordre the boards.
Can i ask you for e référence for the resistors ?

**Reply by darthcloud** [reply]
This board is generic for multiple console, you populate components only if required for the specific one you building.

The cable wiki got all the part listed for each cable in the pcb section (ignore DIY Through-hole section)

**Comment by MoockyLoock** [comment]
Oh sorry.
I may be wrong but it seems that the part number for the resistance, for both NES ans SNES, lead to a through-hole model.
I could search myself for a SMD one but i'm afraid i could choose a wrong one.
Sorry once again for bothering you ^^

**Reply by darthcloud** [reply]
I updated the wiki thanks for reporting.

Btw: https://www.digikey.ca/en/products/detail/yageo/RC0402FR-073K6L/2827641
